commcare-cloud/commcare_cloud.py

```diff
@@ -224,20 +224,64 @@ def make_parser(parser):
         parser.add_argument('-u', '--user', default='ansible', help=(
             "The user to run the commands as."
         ))
+        parser.add_argument('--raw', action='store_true', help=(
+            "Run the raw shell command as given rather than trying to be smart "
+            "about interpreting intent. Currently only 'smart' behavior is replacing sudo "
+            "with --become --ask-become-pass."
+        ))
 
     @staticmethod
     def run(args, unknown_args):
         public_vars = get_public_vars(args.environment)
+        shell_command = args.shell_command
+        become = False
+        become_user = None
+        ask_become_pass = False
+        if shell_command.strip().startswith('sudo '):
+            parts = shell_command.strip().split()
+            if len(parts) > 2 and parts[1] == '-u':
```

Comment:
This is fragile. I sometimes pile options together (like `sudo -iu user`), which would have confusing/wrong results here. Maybe it would be better if it didn't try to be so smart?